internal/testing/check_api_change.sh

```diff
@@ -53,12 +51,16 @@ trap cleanup EXIT
 
 git clone -b "$UPSTREAM_BRANCH" . "$MASTER_CLONE_DIR" &> /dev/null
 
+# Run the following checks in the master directory
+ORIG_DIR=$(pwd)
```

Comment:
Surprised shellcheck didn't catch this:

```suggestion
ORIG_DIR="$(pwd)"
```